x-pack/plugins/security_solution/public/onboarding/common/lib/telemetry.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { METRIC_TYPE, TELEMETRY_EVENT, track } from '../../../common/lib/telemetry';
+
+export const trackOnboardingLinkClick = (linkId: string) => {
+  track(METRIC_TYPE.CLICK, `${TELEMETRY_EVENT.ONBOARDING}${linkId}`);
```

Comment:
NIT: Maybe it would be better to add the separator here `${TELEMETRY_EVENT.ONBOARDING}_${linkId}` instead of having a trailing `_` inside the `ONBOARDING` const?